bldsys/cmake/template/gradle/app.build.gradle.in

```diff
@@ -2,13 +2,14 @@ plugins {
 	id 'com.android.application'
 }
 
-ext.vvl_version='1.3.275.0'
+ext.vvl_version='1.3.296.0'
 apply from: "./download_vvl.gradle"
 
 android {
 	compileSdk 33
```

Comment:
compileSdk should be bumped to 35

Reply:
I had most of those version updates (build tools, NDK) done by Android Studio's assistant. Getting this up and running took me roughly a month so I'd rather not want to change up things again.

Reply:
I'm sorry Sascha, I meant to offer to help.  I went ahead and made a PR to your branch; feel free to accept / test out the changes to get the updates I'm suggesting.  There was one in there that is only deprecated, which means it'll disappear in a future Android Studio release.  might as well solve it now no?

Reply:
No offense taken :)

Merged your PR, so this should be addresed.